pyccel/codegen/printing/pycode.py

```diff
@@ -996,6 +1004,21 @@ def _print_PyccelSymbol(self, expr):
 
     def _print_PythonType(self, expr):
         return 'type({})'.format(self._print(expr.arg))
+    
+    #-----------------Class Printer---------------------------------
+
+    def _print_ClassDef(self, expr):
+        classDefName = 'class {}({}):'.format(expr.name,', '.join(self._print(arg) for arg in  expr.superclass))
+        methods = ''.join(self._print(method) for method in expr.methods)
+        methods = self._indent_codestring(methods)
+        classDef = '\n'.join([classDefName, methods]) + '\n'
+        return classDef
+
+    def _print_ConstructorCall(self, expr):
+        return f"{expr.func.cls_name}({', '.join(self._print(arg) for arg in expr.arguments)})"
```

Comment:
See above, it is not easy to see at a glance what the output string will look like

Reply:
```suggestion
        args = ', '.join(self._print(arg) for arg in expr.arguments)
        return f"{expr.func.cls_name}({args})"
```